sphinx_library/about.html

```diff
@@ -1,17 +1,13 @@
-{%- if logo %}
+{%- if logo_url %}
 <p class="logo">
   <a href="{{ pathto(master_doc)|e }}" title="{{ project }}">
-    {% if '://' in logo %}
-    <img class="logo" src="{{ logo }}" alt="{{ project }}"/>
-    {% else %}
-    <img class="logo" src="{{ pathto('_static/' + logo, 1)|e }}" alt="{{ project }}"/>
-    {% endif %}
+    <img class="logo" src="{{ logo_url }}" alt="{{ project }}"/>
```

Comment:
N.B.: The value of logo_url is correct as-is for either URL or local file references and the prior `_static/` prefixing resulted in `_static/_static/image.png`, which was broken; so simplified here.